lagunita/lms/static/sass/partials/base/custom_overrides/_mobile.scss

```diff
@@ -39,7 +37,24 @@ section.container {
     }
 }
 
-@media (max-width: $screen_width_small) {
+// Phone screens (Phones only)
+@media (max-width: $screen_break_xs_s - 1) {
+    #global-navigation {
+        .user {
+            .user-link {
+                .label-username {
+                    position: absolute;
+                    width: 0;
+                    height: 0;
+                    overflow: hidden;
+                }
+            }
+        }
+    }
+}
+
+// Handheld screens (Phones to small tablets)
+@media (max-width: $screen_break_s_m - 1) {
```

Comment:
Why the subtraction? Do we use this idiom elsewhere?

Reply:
@stvstnfrd the subtraction is to avoid overlap on the specific break-point.

we want XS to be < break-point, not <=